### DISCUSSION TITLE
Provide pre-compiled arm64 binaries

### DISCUSSION THREAD

**Discussion by pyromaniac3010** [CardanoSolutions/ogmios#262] [Archives]
#### Describe your idea, in simple words.
It would be great to have a pre-compiled arm64 binary on the release page available.
#### Why is it a good idea?
Cloud providers offer cheaper priced instances with arm64 processors.
#### Are you willing to work on it yourself?
I could test the arm64 binaries on my infrastructure

**Comment by KtorZ** [comment]
This is indeed in the making. We are looking into adding ARM support for the docker image, which will probably demand that we get static arm64 binary first.

**Comment by theshashankpal** [comment]
Hi @pyromaniac3010 ,
This issue seems to be a good initiation of my journey towards contribution to FOSS projects.
Can you please assign this to me?

**Comment by KtorZ** [comment]
@theshashankpal I am currently experimenting with cross compiling using [Haskell.nix](https://input-output-hk.github.io/haskell.nix/); so far, it's building :crossed_fingers:, I'll let you know how it goes and if it doesn't work, I am more than happy to get some assistance :)

**Comment by KtorZ** [comment]
It didn't work as planned in the end: https://github.com/input-output-hk/haskell.nix/issues/1652

**Comment by pyromaniac3010** [comment]
The arm64 binaries (non docker) could be used to improve the easy installation with https://github.com/cardano-community/guild-operators. I'm currently working on some patches there to make koios work on arm64 instances.
